src/main/java/org/apache/mesos/hdfs/executor/AbstractNodeExecutor.java

```diff
@@ -151,6 +156,8 @@ protected void startProcess(ExecutorDriver driver, Task task) {
         process = Runtime.getRuntime().exec(new String[]{
             "sh", "-c", task.cmd});
         redirectProcess(process);
+        TimedHealthCheck healthCheck = new TimedHealthCheck(driver, task);
```

Comment:
Let's update the timings for the health check so that we wait a reasonable amount of time for DNS to register. We want to avoid letting nodes die and then relaunching them.